docs/Tutorials - start here/semantic_segmentation_tutorial.md

```diff
@@ -0,0 +1,42 @@
+# How to Use Innofw Framework for Semantic Segmentation

```

Comment:
can you provide introduction to what the semantic segmentation is?
What are the input data? What is mask? What is the model output?
What about the dimensions of the input file?
How DataModule for the Segmentation returns batches?(hint, SegDataKeys)
What about the output of the model, how it is structured?(hint, SegOutKeys)

Reply:
Better would be to add some sample images